modules/docs/resources/_meta.js

```diff
@@ -0,0 +1,14 @@
+export default {
```

Comment:
Let's omit this file since it is specific to an internal use case. I think we can brainstorm an alternative approach like housing this internally or something along those lines